var/spack/repos/builtin/packages/xorg-cf-files/package.py

```diff
@@ -17,6 +17,7 @@ class XorgCfFiles(AutotoolsPackage, XorgPackage):
 
     license("custom")
 
+    version("1.0.8", sha256="c6f1c9ffce96278a9d7c72d081e508d81c219dec69ae0dbaf8ae88f4bc9ef977")
```

Comment:
https://github.com/spack/spack/pull/36456 issues with util-macros may still apply? If not, we can close that one...

Reply:
`xorg-cf-files` appears to build correctly for me without `util-macros@1.20`, but we can pull this out of this PR if you'd like to take a closer look.

Reply:
Let me try this quickly in a codespaces cleanroom environment.

Reply:
On configure, we get:
```
Package xorg-macros was not found in the pkg-config search path.
Perhaps you should add the directory containing `xorg-macros.pc'
to the PKG_CONFIG_PATH environment variable
Package 'xorg-macros' not found
```
but still builds and installs. Same warning appears in an install of 1.0.6, and installed fileset is the same, so this is not a regression.

Adding `util-macros` (most recent version 1.19.3) as build dependency to 1.0.6 causes it to be picked up, no warning anymore, but same fileset upon installation (only diffs in `.spack/` and in `lib/X11/config/site.def` where ProjectRoot differs).

So, I'm happy to resolve this and close #36456 since I don't think the `util-macros` dependency appears needed.